qa/655

```diff
@@ -0,0 +1,114 @@
+#! /bin/sh
+# PCP QA Test No. 655
+# test llc PMDA
+#
+# Copyright (c) 2023 Red Hat.  All Rights Reserved.
+#
+
+seq=`basename $0`
+echo "QA output created by $seq"
+
+# get standard environment, filters and checks
+. ./common.product
+. ./common.filter
+. ./common.check
+
+case "$PCP_PLATFORM"
+in
+    linux)
+    	;;
+    *)
+	_notrun "No llc support for PCP_PLATFORM $PCP_PLATFORM"
+	# NOTREACHED
+	;;
+esac
+
+
+_cleanup()
+{
+    [ -f $tmp.conf.backup ] && $sudo cp $tmp.conf.backup $PCP_DIR/etc/pcp.conf
+    _restore_pmda_install $iam >$here/$seq.full 2>&1
+    rm -f $tmp.*
+    exit $status
+}
+
+_check()
+{
+    pminfo -f llc > $tmp.pminfo
+
+    PCP_TMPFILE_DIR=/tmp pcp atop -P LLC 1 3 > $tmp.pcpatop
+
+    $PCP_AWK_PROG '
+    /llc.occupancy/ {getline;gsub("[0-9.]+",0.99);print "llc.occupancy",$6} 
+    /llc.mbm_local/ {type=$0;getline;gsub("[0-9.]+",999);print "llc.mbm_local",$6} 
+    /llc.mbm_total/ {getline;gsub("[0-9.]+",999);print "llc.mbm_total",$6}' $tmp.pminfo
+
+    $PCP_AWK_PROG '
+    /LLC / {gsub("[0-9]+","999",$3);gsub("[0-9]+","999",$6);gsub("[0-9.]+","99",$8);print $3,$6,$8}' $tmp.pcpatop
+}
+
+if [ ! -d /sys/fs/resctrl/mon_data ] ; then
+for d in /tmp/sys/fs/resctrl/mon_data/mon_L3_00/ ; do
```

Comment:
Use $tmp over /tmp?  (this way we can easily tell which QA test wrote the tmp files)

Do we clean these up at the end BTW?  Looks like that logic is missing in _cleanup.